src/pages/feedback/send-feedback/send-feedback.ts

```diff
@@ -121,14 +121,12 @@ export class SendFeedbackPage {
       'appreciate-review'
     );
     infoSheet.present();
-    infoSheet.onDidDismiss(async () => {
+    infoSheet.onDidDismiss(async option => {
+      if (!option) return;
       if (this.launchReview.isRatingSupported()) {
-        this.launchReview.rating().then(val => {
-          if (val == 'dismissed') this.navCtrl.popToRoot({ animate: false });
-        });
+        this.launchReview.rating();
       } else {
         await this.launchReview.launch();
-        this.navCtrl.popToRoot({ animate: false });
```

Comment:
Why we should keep in that page? If users back to the app, they 'll see feedback page again. I 'm not sure if it 's a good UX.